pkg/config/apiserver.go

```diff
@@ -88,6 +104,58 @@ func (a *API) register(c *restful.Container) {
 		Param(ws.PathParameter("subject", "subject").DataType("string")).
 		Reads(&pb.ServiceConfig{}).
 		Writes(APIResponse{}))
+
+	// Creates or replaces a rule's list of aspects
+	ws.Route(ws.
+		PUT("/scopes/{scope}/subjects/{subject}/rules/{ruleid}/aspects/{aspect}").
+		To(a.putAspect).
+		Doc("Creates or replaces a rule’s list of aspects.").
+		Param(ws.PathParameter("scope", "scope").DataType("string")).
+		Param(ws.PathParameter("subject", "subject").DataType("string")).
+		Param(ws.PathParameter("aspect", "aspect").DataType("string")).
```

Comment:
Should this capture `ruleid` as a param too? Also, it looks like you use underscores in other places, here and elsewhere should the `ruleid` param name be `rule_id` instead?